tests/test_ssl.py

```diff
@@ -101,6 +101,23 @@
 skip_if_py3 = pytest.mark.skipif(PY3, reason="Python 2 only")
 
 
+def socket_any_family():
+    try:
+        return socket(AF_INET)
+    except error as e:
+        if e.errno == EAFNOSUPPORT:
+            return socket(AF_INET6)
+        raise e
```

Comment:
```suggestion
        raise
```

this will preserve the stack trace better.